src/features/tokens/routes/fetch.ts

```diff
@@ -0,0 +1,159 @@
+import { ProtocolType } from '@hyperlane-xyz/sdk';
```

Comment:
Mostly just moved here from `routes/hooks.ts` with a few tweaks to have it avoid creating duplicate routes between collateral tokens